upup/models/cloudup/resources/addons/networking.cilium.io/k8s-1.7.yaml.template

```diff
@@ -121,6 +121,9 @@ data:
   install-iptables-rules: "{{- if .IPTablesRulesNoinstall -}}false{{- else -}}true{{- end -}}"
   auto-direct-node-routes: "{{- if .AutoDirectNodeRoutes -}}true{{- else -}}false{{- end -}}"
   enable-node-port: "{{- if .EnableNodePort -}}true{{- else -}}false{{- end -}}"
+  {{ if .EnableNodePort }}
+  kube-proxy-replacement: strict
+  {{ end }}
```

Comment:
```suggestion
  kube-proxy-replacement: "{{- if .EnableNodePort -}}strict{{- else -}}probe{{- end -}}"
```

Reply:
I would perhaps go for `kube-proxy-replacement: disabled`. it is a bit easier to understand what is happening if one is explicit.

Reply:
Sounds even better to me.